### ISSUE TITLE
Extra space in keyword expansion

### ISSUE THREAD

**Issue by r39ex** [jrockway/cperl-mode#33]
With cperl-electric-keywords set, when typing if + space I get an expanded construct like this (_ is point):

```
if ( _) {
}
```
1. Is this intended behaviour?
2. If so, is there a setting or some other way to get rid of the extra space before point?

**Comment by hakonhagland** [comment]
Note that according to [perlstyle](https://metacpan.org/pod/distribution/perl/pod/perlstyle.pod) spaces after the opening parenthesis is recommended for readability. For example if I run [perltidy](https://metacpan.org/pod/distribution/Perl-Tidy/bin/perltidy) on this code:

```
if($foo==3) {
    say "Hello";
}
```

it will insert extra spaces like this:

```
if ( $foo == 3 ) {
    say "Hello";
}
```

**Comment by r39ex** [comment]
Yes, but timtowtdi =). That's why perltidy has an option like  --paren-tightness. I would personally write it like this:
```
if ($foo == 3) {
    say "Hello";
}
```

**Comment by renormalist** [comment]
I don't think someone will fix this, and tend to close the issue unless someone objects.
